lib/robots/dor_repo/ocr/stage_files.rb

```diff
@@ -20,6 +20,8 @@ def move_files(workspace_dir)
           content_dir = File.join(workspace_dir, 'content')
 
           abbyy_results = Dor::TextExtraction::Abbyy::Results.find_latest(druid:, logger:)
+          raise 'No ABBYY Result XML file found' unless abbyy_results
+
```

Comment:
this shouldn't happen (i.e. the result XML file should be still be there) but if for some reason none are found, this gives us a better error message than what would happen now (which is this: `NoMethodError: undefined method 'move_result_files' for nil abbyy_results.move_result_files(content_dir)` (https://app.honeybadger.io/projects/52894/faults/108106645/01J0YEE9S295KVY5ZTYWAQ3CMJ?page=0)